src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/CreateInvitation/CreateInvitationCommand.cs

```diff
@@ -8,8 +8,18 @@ public class CreateInvitationCommand : IAsyncRequest
     {
         public string ExternalUserId { get; set; }
         public string HashedAccountId { get; set; }
+
+        /// <summary>
+        /// The person being invited
+        /// </summary>
         public string Name { get; set; }
+        /// <summary>
+        /// The person being invited
```

Comment:
duplicated descriptions, need to put the correct values in

Reply:
Replaced by better property names